cmd/cmd.go

```diff
@@ -715,19 +704,6 @@ func generateInteractive(cmd *cobra.Command, model string) error {
 	}
 }
 
-func generateBatch(cmd *cobra.Command, model string) error {
```

Comment:
Maybe we could have a doc which explains how to run this as a batch?

Reply:
Will add this to the PR (and to docs if folks look for it):

```
while IFS= read -r line; do
    echo "$line" | ollama run $1
done
```

Usage:

```
cat questions.txt | ./batch.sh llama2
```